pkgs/by-name/on/onthespot/package.nix

```diff
@@ -27,6 +28,7 @@ python3.pkgs.buildPythonApplication rec {
   propagatedBuildInputs = with python3.pkgs; [
     charset-normalizer
     defusedxml
+    ffmpeg
```

Comment:
Is ffmpeg required during build? Or at runtime?

Reply:
Runtime

Reply:
The most correct way would probably be to add  `--prefix PATH : ${lib.mkBinPath [ ffmpeg ]}` to `makeWrapperArgs`, but I believe this has the same side effect. I don't think it would hurt any dependent packages (if any) to have ffmpeg propagated.

Reply:
I would prefer ’--prefix PATH : ${lib.makeBinPath [ ffmpeg ]}’ since this is what I know is typically used in this case. What do you think?